assets/javascripts/lib/uploadHandler.js.es6

```diff
@@ -0,0 +1,48 @@
+import { isImage } from "discourse/lib/uploads";
```

Comment:
Nitpick: I believe this file should be named `upload-helpers.js.es6` to keep the filenames consistent.